Cargo.toml

```diff
@@ -4,57 +4,47 @@ version = "0.1.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
 
+# Ensure all dependencies + feats are mapped to crate features for correct usage
+# default features often have std breaking no_std and potentially other unwanted
 [dependencies]
-aead = "0.5.2"
-aes-gcm = "0.10.3"
-block-buffer = "0.10.4"
-chacha20poly1305 = "0.10.1"
-crypto-common = "0.1.6"
-der = "0.7.8"
-digest = "0.10.7"
-ecdsa = "0.16.8"
-ed25519-dalek = { version = "2.0.0", features = ["pkcs8"] }
-elliptic-curve = "0.13.5"
-generic-array = "0.14.7"
-hmac = "0.12.1"
-p256 = { version = "0.13.2", features = ["ecdh"] }
-p384 = { version = "0.13.0", features = ["ecdh"] }
-paste = "1.0.14"
-pkcs1 = { version = "0.7.5", features = ["pem", "pkcs8"] }
-pkcs8 = { version = "0.10.2", features = ["pem", "pkcs5"] }
+aead = { version = "0.5.2", default-features = false }
+aes-gcm = { version = "0.10.3", default-features = false, features = ["aes", "alloc"] }
+chacha20poly1305 = { version = "0.10.1", default-features = false }
+crypto-common = { version = "0.1.6", default-features = false }
+der = { version = "0.7.8", default-features = false }
+digest = { version = "0.10.7", default-features = false }
+ecdsa = { version = "0.16.8", default-features = false, features = ["alloc"] }
+ed25519-dalek = { version = "2", default-features = false, features = ["pkcs8"] }
+hmac = { version = "0.12.1", default-features = false }
+# TODO: figure out why std was needed here
```

Comment:
What's up with this?

Reply:
ah artefact when I was tackling that ecdsa crate needed the alloc feature for one implementation - removed thx!